src/main/java/com/google/sps/data/PersonDao.java

```diff
@@ -0,0 +1,24 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps;
+
+/* PersonDao includes the basic methods anything managing Person entities must support */
+public interface PersonDao {
+  // Returns a Person object from storage, with email as the key.
+  public Person get(String email) throws com.google.appengine.api.datastore.EntityNotFoundException;
```

Comment:
Yeah I think you're aware, but PersonDao shouldn't be throwing an exception specific to one implementation of the interface.

(That said, I've got a possible remedy for this as a comment in the other file)

Reply:
This hasn't been fixed.